### PR TITLE
[[, [: add/update pages

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/[.md [hunk 1]
```diff
@@ -2,24 +2,24 @@
...
 > Check file types and compare values.
 > Returns 0 if the condition evaluates to true, 1 if it evaluates to false.
-> More information: <https://www.gnu.org/software/coreutils/test>.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#Conditional-Constructs>.
 
-- Test if a given variable is equal to a given string:
+- Test if two strings are equal/not equal:
 
-`[ "{{$VARIABLE}}" == "{{/bin/zsh}}" ]`
+`[ "${{variable}}" {{==|=!}} "{{string}}" ]`
```
Comment (258204) [inline pages/common/[.md:9]:
```suggestion
`[ "${{variable}}" {{==|!=}} "{{string}}" ]`
```

File: pages/common/[.md [hunk 2]
```diff
@@ -2,24 +2,24 @@
 
 > Check file types and compare values.
 > Returns 0 if the condition evaluates to true, 1 if it evaluates to false.
-> More information: <https://www.gnu.org/software/coreutils/test>.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#Conditional-Constructs>.
 
-- Test if a given variable is equal to a given string:
+- Test if two strings are equal/not equal:
```
Comment (258204) [inline pages/common/[.md:7]: The original makes more sense since you are moving the $ outside the {{}}

File: pages/common/[[.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# [[
+
+> Check file types and compare values.
+> Returns 0 if the condition evaluates to true, 1 if it evaluates to false.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#Conditional-Constructs>.
+
+- Test if two strings are equal/not equal:
+
+`[[ ${{variable}} {{==|=!}} "{{string}}" ]]`
```
Comment (258204) [inline pages/common/[[.md:9]:
```suggestion
`[[ ${{variable}} {{==|!=}} "{{string}}" ]]`
```

File: pages/common/[[.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Test if a string conforms the specified glob/regex:
+
+`[[ ${{variable}} {{==|=~}} {{pattern}} ]]`
+
+- Test if two numbers are [eq]ual/[n]ot [e]qual/[g]reater [t]han/[l]ess [t]han/[g]reater than or [e]qual/[l]ess than or [e]qual:
+
+`[[ ${{variable}} -{{eq|ne|gt|lt|ge|le}} "{{string}}" ]]`
+
+- Test if a the specified [f]ile exist:
```
Comment (258204) [inline pages/common/[[.md:19]:
```suggestion
- Test if the specified [f]ile exists:
```

File: pages/common/[[.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+`[[ -f {{path/to/file}} ]]`
+
+- Test if a the specified [d]irectory exist:
```
Comment (258204) [inline pages/common/[[.md:23]:
```suggestion
- Test if the specified [d]irectory exists:
```

File: pages/common/[[.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+`[[ -d {{path/to/directory}} ]]`
+
+- Test if a the specified file or directory [e]xist:
```
Comment (258204) [inline pages/common/[[.md:27]:
```suggestion
- Test if the specified file or directory [e]xists:
```

**Review by 258204 — CHANGES_REQUESTED** [review]
Good work!

File: pages/common/[.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply pages/common/[.md]: I've updated descriptions to be more user-friendly. ;)

**Review by 258204 — COMMENTED** [review]
Thanks @EmilySeville7cfg !

**Review by 258204 — APPROVED** [review]

File: pages/common/[.md [hunk 7]
```diff
@@ -2,24 +2,24 @@
...
-- Test if a given variable is equal to a given string:
+- Test if a given variable is equal/not equal to the specified string:
 
-`[ "{{$VARIABLE}}" == "{{/bin/zsh}}" ]`
+`[ "${{variable}}" {{==|!=}} "{{string}}" ]`
 
-- Test if a given variable is empty:
+- Test if a given variable is [eq]ual/[n]ot [e]qual/[g]reater [t]han/[l]ess [t]han/[g]reater than or [e]qual/[l]ess than or [e]qual to the specified number:
 
-`[ -z "{{$GIT_BRANCH}}" ]`
```
Comment (sbrl) [inline pages/common/[.md:13]: This example is useful. I'm not sure I agree with removing it.

Comment (EmilySeville7cfg) [reply pages/common/[.md:13]: I can recover it as a separate code snippet. But if I were you I would place this snippet in [Jenkins](https://wiki.jenkins.io/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-belowJenkinsSetEnvironmentVariables) tldr page.

**Review by blueskyson — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @EmilySeville7cfg!

**Pull request merged**